examples/js/loaders/GLTF2Loader.js

```diff
@@ -2789,7 +2789,9 @@ THREE.GLTF2Loader = ( function () {
 										break;
 
 									default:
+										var previousDrawMode = child.drawMode;
 										child = new THREE.Mesh( originalGeometry, material );
+										child.drawMode = previousDrawMode;
```

Comment:
How about doing `var originalChild = child;` at the beginning of the for loop and then doing:

```js
child.drawMode = originalChild.drawMode;
```